Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,133 @@
+
+<sup>2</sup> The system will be located outdoors, so it is essential that the electrical components will be protected from environemental weather hazards and damage. The IP67 rating protects the system from cover rain, snow, dust, wind, and UV. Using this NEMA box will improve the longevity and prevent damage of the system. <br />
+
+<sup>3</sup> This constraint is most important for system functionality, because the power system must supply the rated power for the camera system to acheive full functionality without running into any under voltage scenarios that could impact system peformance. Refer to Table 2 for the full list of power requirements of the system.  <br />
+
+<sup>4</sup> This constraint ensures that the power supply will not be damaged by an induced back EMF from the servo motors. 
+<br />
+
+<sup>5</sup> This constraint ensures safety of the system by providing a "kill switch" to immediately remove the power supplied to both power supplies in an emergency scenario.
+<br />
+
+<sup>6</sup> This constraint ensures the safety of the system's user and the location's property. Especially with the usage of wall power, safety measures must be taken to prevent hazards like shocks and fires. For shock protection, the system will need to be properly grounded and insulated for the user's safety from electric shock. For overcurrent proection, the system will need to be connected to a circuit breaker to prevent overheating and fires due to excessive current flow. 
+<br />
 
-<sup>3</sup> Reason for contraint 3 and why it is important
 
 ## Schematic
 
+![Screenshot 2024-09-23 at 9 02 50 AM](https://github.com/user-attachments/assets/f3f163ac-4551-47ce-9042-146f410d8984)
+<br />
+Figure 1: Camera Power Buildable Schematic.<br />
+The above schematic shows the component connections for the system. We will be utilizing a wall outlet for this design then using a 15 ft two way extensions cord with switching functionality. One side of the extension powers the Raspberry Pi 5 V 15 W USB-C power supply while the other powers the DC 5 V 10 A 50 W power supply. The Raspberry Pi 5 V 15 W USB-C power supply provides power to the Raspberry Pi 4B then a CSI ribbon cable is used to provide power at 5 V from the Raspberry Pi 4B to the Raspberry Pi HQ camera. To prevent damage from back EMF produced by the servo motors, a 1N5400 diode is placed across the terminals of both motors.
+<br />
+
+
+
 ## Analysis
+| Component | Voltage (V)| Min Current (A)| Max Current (A)| Min Power (W)| Max Power (W)|
```

Comment:
1. It would be good to have some text at the beginning of this section rather than starting out with a table.
2. Incorrect units used for torque.
3. The achievement of constraint 6 is not clear (except for the related wire gauge topic). How is this achieved?